### PR TITLE
Fix lost patch and remove mischange

### PR DESCRIPTION
**Pull request by joeyleeeeeee97**
In https://github.com/AdoptOpenJDK/openjdk-build/pull/2232,  I accidentally overrode the branch with some other test changes, which lead to some 'unreviewed and unexpected code' into the master.
I am really sorry about that, I would use a separate patch every pr.

I created a new patch reverting all changes and reapply that patch, sorry for the confusion and hopefully didn't cause any trouble.
Issue: https://github.com/AdoptOpenJDK/openjdk-build/issues/2292

### CODE REVIEW

File: pipelines/build/common/openjdk_build_pipeline.groovy [hunk 1]
```diff
@@ -1144,13 +1145,15 @@ class Build {
...
+                    try {
+                        context.timeout(time: buildTimeouts.SIGN_JOB_TIMEOUT, unit: "HOURS") {
+                            sign(versionInfo)
+                        }
+                    } catch (FlowInterruptedException e) {
+                        context.println "[ERROR] Sign job timeout (${buildTimeouts.SIGN_JOB_TIMEOUT} HOURS) has been reached. Exiting..."
+                        throw new Exception()
                     }
-                } catch (FlowInterruptedException e) {
-                    context.println "[ERROR] Sign job timeout (${buildTimeouts.SIGN_JOB_TIMEOUT} HOURS) has been reached OR the downstream sign job failed. Exiting..."
```
Comment (M-Davies) [inline pipelines/build/common/openjdk_build_pipeline.groovy:1152]: In your rebase, did you pull in the latest commits as I don't think this relates to the commit's you're trying to revert?

**Review by M-Davies — COMMENTED** [review]
What is the change you're trying to revert? Is it [this one](https://github.com/AdoptOpenJDK/openjdk-build/pull/2293/files?diff=unified&w=1#diff-26504d2cca33fa01eba42bae86b78103384f552dfefc8c5ca32994f947c8ee83)?

EDIT: Ah I see there's some docker file changes in there too. Assuming my comment below is valid, I'm ok with this going through

File: pipelines/build/common/openjdk_build_pipeline.groovy (hunk 1 above)
Comment (joeyleeeeeee97) [reply pipelines/build/common/openjdk_build_pipeline.groovy:1152]: yes. this is not relevent. Corrected.

**Review by M-Davies — APPROVED** [review]
LFTM

**Review by karianna — APPROVED** [review]
LGTM

**Pull request merged**